include/boost/geometry/extensions/gis/io/wkb/detail/parser.hpp

```diff
@@ -117,7 +118,9 @@ struct geometry_type_parser
     }
 };
 
-template <typename P, int I, int N>
+template <typename P,
+          std::size_t I = 0,
+          std::size_t N = dimension<P>::value>
 struct parsing_assigner
```

Comment:
Indentation is changed. If so, please follow conventions then, angle-bracket new line, last one too